integration-webcert/src/test/java/se/inera/intyg/minaintyg/integration/webcert/converter/data/CertificateDataConverterTest.java

```diff
@@ -399,6 +400,51 @@ void shallReturnErrorValueWhenValueConverterDoesntExists() {
     assertEquals(expectedCategories, actualCategories);
   }
 
+  @Test
+  void shallExcludeQuestionsOfTypeMessageFromConversion() {
+    final var expectedCategories = List.of(
+        createCertificateCategory(CAT_ONE_TEXT,
+            createCertificateQuestion(QN_ONE_TEXT, null, QN_TEXT_VALUE)
+        )
+    );
+
+    final var elements = List.of(
+        createCategoryElement(CAT_ONE_TEXT, CAT_ONE_ID, 0),
+        CertificateDataElement.builder()
```

Comment:
If you don't need to test specific values you can use: createQuestionElement(QN_ONE_TEXT, "", QN_ONE_ID, 1, CAT_ONE_ID)